entity-framework/core/miscellaneous/cli/dotnet.md

```diff
@@ -70,7 +70,7 @@ Run the following commands to verify that EF Core CLI tools are correctly instal
 
 The output from the command identifies the version of the tools in use:
 
-```console
+```output
```

Comment:
consistency issue in output vs Output

Reply:
What's inconsistent? Console is for commands you can run in a console; output is for the output of console applications.

Reply:
first letter capital/non-capital in different places.

Reply:
Ah, fixed.